tests/tracker_test.go

```diff
@@ -1,9 +1,29 @@
 package easypost_test
 
 import (
+	"strconv"
+
 	"github.com/EasyPost/easypost-go"
 )
 
+func (c *ClientTests) TestTrackerCreateList() {
```

Comment:
Usually when adding something to an already existing file, you append to the end. This test could be at the end.